docs/benchmarks.md

```diff
@@ -59,14 +59,15 @@ for `spark.default.parallelism`, which by default is based on the number of avai
 cores. This value can be set explicitly to better control the size of the output files.
 
 It should also be noted that no decimal types will be output. The conversion code uses explicit 
-schemas to ensure that decimal types are converted to floating-point types instead.
+schemas to ensure that decimal types are converted to floating-point types instead because the
+plugin does not yet support decimal types but these will be supported in a future release.
 
-## Running Benchmarks
+## Running Benchmarks from a Spark shell
 
 The benchmarks can be executed in two modes currently:
 
 - Execute the query and collect the results to the driver
-- Execute the query and write the results to disk (in Parquet or CSV format)
+- Execute the query and write the results to disk (in Parquet or ORC format)
```

Comment:
So, this wasn't an error after all. The benchmarks do currently support writing. to Parquet and CSV only. I will add ORC support as part of this PR.